tests/models/test_evc_base.py

```diff
@@ -2,16 +2,14 @@
 import sys
 from unittest import TestCase
 
-from kytos.core.interface import UNI
+from napps.kytos.mef_eline.models import EVC
+from napps.kytos.mef_eline.scheduler import CircuitSchedule
+from tests.helpers import get_uni_mocked, get_controller_mock
 
 # pylint: disable=wrong-import-position
 sys.path.insert(0, '/var/lib/kytos/napps/..')
```

Comment:
@rmotitsuki, `sys.path.insert` must stay before the `napps` import, or it loses its function.

Reply:
OK. fixed